_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -97560,7 +97556,6 @@
 	name = "Medbay Maintenance";
 	req_access_txt = "5"
 	},
```

Comment:
Is... This a decal with a name and required access level?

Reply:
Nah I've just looked and it's just a quirk with the git display, the only thing named "medbay maintenance" is the maint door in medbay.